code/api/src/modules/user/tests/userQuery.test.js

```diff
@@ -17,13 +17,16 @@ describe('User queries', () => {
 
         res.body.data.user.should.have.property('email')
         expect(res.body.data.user.email).to.eq('user@crate.com')
+
+        res.body.data.user.should.have.property('description')
+        expect(res.body.data.user.description).to.eq('Tattooed seitan waistcoat austin asymmetrical chambray hot chicken man bun poke')
         done();
     })
   });
 
   it('gets information for all users', (done) => {
     request.post('/graphql')
-    .send({ query: '{ users { image email } }'})
+    .send({ query: '{ users { image email description} }'})
```

Comment:
I also like that we aren't just testing GETing a single user and checking the attribute, but also AllUsers